src/System.Management.Automation/engine/Modules/ImportModuleCommand.cs

```diff
@@ -1941,6 +1962,56 @@ private List<T> FilterModuleCollection<T>(IEnumerable<T> moduleCollection)
             return filteredModuleCollection;
         }
 
+        private void PrepareNoClobberWinCompatModuleImport(string moduleName, ModuleSpecification moduleSpec, ref ImportModuleOptions importModuleOptions)
+        {
+            List<string> NoClobberModuleList = PowerShellConfig.Instance.GetWindowsPowerShellCompatibilityNoClobberModuleList();
+
+            // moduleName can be just a module name and it also can be a full path to psd1 from which we need to extract the module name
+            var coreModuleToLoad = moduleName;
+            if (moduleSpec != null)
+            {
+                coreModuleToLoad = moduleSpec.Name;
+            }
+            else if (Path.IsPathRooted(moduleName))
+            {
+                coreModuleToLoad = Path.GetFileNameWithoutExtension(moduleName);
+            }
+
+            var coreModuleToLoadIsEngineModule = InitialSessionState.IsEngineModule(coreModuleToLoad);
```

Comment:
```suggestion
            bool isModuleToLoadEngineModule = InitialSessionState.IsEngineModule(coreModuleToLoad);
```

Reply:
Reading the variable name, I see `coreModuleToLoad` and think this is an object, and then see `IsEngineModule` and the [garden path](https://en.wikipedia.org/wiki/Garden-path_sentence) turns to boolean. Starting the variable name with `is` primes me for a boolean

Reply:
renamed.